versions.props

```diff
@@ -1,6 +1,6 @@
 com.diffplug.spotless:spotless-plugin-gradle = 6.4.0
 com.google.auto.service:auto-service = 1.0.1
-com.google.errorprone:error_prone_* = 2.11.0
+com.google.errorprone:* = 2.11.0
```

Comment:
What does this buy us? It may confuse our version upgrade excavators

Reply:
How so?

Reply:
As long as our excavators don't try to upgrade `com.google.errorprone:javac` (which I don't think they do), this should be fine.

Reply:
Why introduce the potential problem?